oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -339,6 +344,36 @@ public boolean updateAttributes() {
         return true;
     }
 
+    /**
+     * If some details couldn't be read from ProcPath.PID_STATUS try reading it from
+     * ProcPath.PID_STAT
+     *
+     * @param status
+     *            status map to fill.
+     * @param stat
+     *            string to read from.
+     */
+    private static void getMissingDetails(Map<String, String> status, String stat) {
+        if (status == null || stat == null) {
+            return;
+        }
+
+        int nameStart = stat.indexOf("(");
```

Comment:
Scrutinzer says using the character `'('` is faster than `"("`.  I'm not sure it matters but why not? :)